Add support to notify users of dependency updates

eerhardt opened the pull request:
UpdateDependencies.ps1 now supports adding a semi-colon delimited list of GitHub users to notify on the PRs it creates.

The actual list of people will be filled out in the 'subscriptions.json' file:  https://github.com/eerhardt/versions/blob/master/Subscriptions.json#L24  (note the long term home for that repro is still TBD)

@weshaggard @stephentoub

stephentoub commented:
LGTM

weshaggard commented:
LGTM as well. Are you planning to add the initial set in subscriptions.json or do you want us to do it?

eerhardt commented:
> Are you planning to add the initial set in subscriptions.json or do you want us to do it?

I've added the initial set here:  https://github.com/eerhardt/versions/blob/master/Subscriptions.json#L29.

For now, I've also added @stephentoub and @weshaggard as contributors to the eerhardt/versions repo, so you can make changes if necessary. I can add more. Just let me know. Hopefully it is short lived and we can get the real place up and running soon.
